test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -233,7 +233,7 @@ var _ = Describe("[performance] Checking IRQBalance settings", func() {
 					return false
 				}
 				return bannedCPUs.IsEmpty()
-			}).WithTimeout(5*time.Minute).WithPolling(10*time.Second).ShouldNot(BeTrue(), "banned CPUs %v not empty on node %q", bannedCPUs, targetNode.Name)
+			}).WithTimeout(5*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "banned CPUs %v not empty on node %q", bannedCPUs, targetNode.Name)
```

Comment:
yes, the intent of the test should actually what you are fixing.
IOW the original boolean condition is inverted, apparently this was not caught during the review, and most worringly I'm not sure why this didn't fail constantly since then.

in #444 we merged code to make sure that `/etc/sysconfig/orig_irq_banned_cpus` always contains empty mask.
when crio starts with highperformace hooks enabled, it unconditionally restores the irqbalance config with the content of `/etc/sysconfig/orig_irq_banned_cpus`. In turn `/etc/sysconfig/orig_irq_banned_cpus` is created if missing the first time crio starts, with no control whatsoever on our side. This is why in #444 we effectively tamper it to make sure the content is predictable.

Since we are fixing this issue, let's also fix the error text, something like
```
"expected empty banned  CPUs on node %q, got %v instead". targetNode.Name, bannedCPUs,
```